lib/Models/GeoJsonCatalogItem.js

```diff
@@ -448,12 +441,15 @@ function loadGeoJson(geoJsonItem) {
         markerSize : defaultValue(parseMarkerSize(style['marker-size']), 20),
         markerSymbol: style['marker-symbol'], // and undefined if none
         markerColor : defaultColor(style['marker-color'], pointPalette),
-        strokeWidth : defaultValue(style['stroke-width'], 2),
+        strokeWidth : defaultValue(style['stroke-width'], 4),
         stroke: defaultColor(style.stroke, lineAndFillPalette),
         markerOpacity: style['marker-opacity'] // not in SimpleStyle spec or supported by Cesium but see below
     };
 
-    options.fill = defaultColor(style.fill, options.stroke.clone);
+    options.fill = defaultColor(style.fill, options.stroke.clone());
+    if (!defined(style['fill'])) {
+        options.fill = options.fill.brighten(0.2, options.fill); 
```

Comment:
Arthur's recently-merged pull request now makes polygon borders black, regardless of their fill color (unless overridden of course).  Maybe check that out and see if you still want to switch it to a lighter version of the fill color.